src/library_nodefs.js

```diff
@@ -25,7 +25,8 @@ mergeInto(LibraryManager.library, {
         "{{{ cDefine('O_RDWR') }}}": flags["O_RDWR"],
         "{{{ cDefine('O_DSYNC') }}}": flags["O_SYNC"],
         "{{{ cDefine('O_TRUNC') }}}": flags["O_TRUNC"],
-        "{{{ cDefine('O_WRONLY') }}}": flags["O_WRONLY"]
+        "{{{ cDefine('O_WRONLY') }}}": flags["O_WRONLY"],
+        "{{{ cDefine('O_NOFOLLOW') }}}": flags["O_NOFOLLOW"]
```

Comment:
Add a trailing comma on the final element too... to help keep diff small when adding new elements line this.